crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -210,28 +234,31 @@ async fn combined_protocol_fees(web3: Web3) {
     .await
     .unwrap();
 
-    let [market_quote_after, limit_quote_after] =
-        futures::future::try_join_all([&market_order_token, &limit_order_token].map(|token| {
-            get_quote(
-                &services,
-                onchain.contracts().weth.address(),
-                token.address(),
-                OrderKind::Sell,
-                sell_amount,
-                quote_valid_to,
-            )
-        }))
+    let [market_quote_after, limit_quote_after, fee_exempt_token_after] =
```

Comment:
Why do we need `fee_exempt_token_after`? Is it expected to be changed?

Reply:
@squadgazzz the idea is to verify that a trader whose address is whitelisted won't have protocol fees applied. Can I reuse the current tokens?

Reply:
I mean, there is `fee_exempt_quote_before ` already. Quote `after` is required only for orders where liquidity pool rebalance is essential. I believe you can use the first quote without requesting the second one.